### PR TITLE
make finding images work in docker dev and ptvsd setup

### PR DESCRIPTION
**Pull request by valentijnscholten**
Partial fix for #2045, makes finding images work in docker development setup.

Making it work in release mode will have to wait until @devGregA has finalized the migration to AppVeyor.

### CODE REVIEW

File: nginx/nginx.conf [hunk 1]
```diff
@@ -25,6 +25,9 @@ http {
     location /static {
       alias /usr/share/nginx/html/static;
     }
+    location /media {
+        alias /usr/share/nginx/html/media;
+    }
```
Comment (alles-klar) [inline nginx/nginx.conf]: This nginx.conf is used in many setups. So I don't think it is a good idea to add it here and only provide a solution for docker-compose.

Comment (valentijnscholten) [reply nginx/nginx.conf]: I couldn't find any other place it is being used

Comment (ptrovatelli) [reply nginx/nginx.conf]: @alles-klar are you thinking of kubernetes? i don't have a strong opinion on this, but if the images are already not working, this is a start... :)

Comment (alles-klar) [reply nginx/nginx.conf:30]: Sorry for the long delay. I think all setups are using the nginx config - also the productive ones. If we add the /media path there, all data that is stored there is free accessible without any authentication. If user add confidential pictures this is a security issue.

Comment (valentijnscholten) [reply nginx/nginx.conf:30]: the production docker-compose setup is a container purely meant for defect dojo. if somebody would  put sensitive data in there I don't think they know what they are doing :-). Also the production (release) config doesn't map any local files to the nginx media folders.  `setup.bash` doesn't use this `nginx.conf`.

Comment (alles-klar) [reply nginx/nginx.conf:30]: Using this setup just for debugging and development is fine but I am afraid that someone adopt this solution for other setups. Maybe we add a comment in the nginx config that it is currently for development only with docker-compose only and that this directory has to be empty in production?

**Pull request merged**